README.md

```diff
@@ -9,11 +9,11 @@ Two different technologies are used to create both sites:
 
 Both parts are merged to a single directory during the CI process.
 
-Node.js and npm are required to build the documentation.
+Node.js(v14 - v16) and npm are required to build the documentation.
```

Comment:
Why do you think we are only compatible to Node v14 to 16?
Our [PR validation](https://github.com/apache/streampipes-website/blob/dev/.github/workflows/pr-validation.yml#L16) and [deployment](https://github.com/apache/streampipes-website/blob/dev/.github/workflows/deploy-website.yml#L16) both run in Node version 18